cmd/cli/version.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright © 2023 SIL International
+*/
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func SetupVersionCmd(parentCommand *cobra.Command) {
+	parentCommand.AddCommand(versionCmd)
+}
+
+var version = "(unknown)"
```

Comment:
Maybe add a comment that the actual value is set during the build process by goreleaser.